scripts/id_table_generator.py

```diff
@@ -3,17 +3,28 @@
 from os import listdir
 import string
 import json
+
+exceptions = [{"original":"estmary", "replacement":"lstmary"}]
+
+def apply_exception(name):
```

Comment:
I think it would be better if you had a make_username function that took first and last name, and applied exceptions internally. IOW, let one function do all the work.